src/client.ts

```diff
@@ -80,6 +96,7 @@ class Client {
 
     this._clientWindow.addEventListener('message', this._onWindowMessage);
     this._clientWindow.addEventListener('click', this._onWindowClick);
+    this._sendToHost(Lifecycle.startedMessage);
```

Comment:
trying out a new way to reference messages, since it was really annoying to have the structure here.  Seemed a lot simpler this way.